Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -211,6 +304,85 @@ void NetKDRequestDevice::LogError(ErrorType error_type, s32 error_code)
   m_scheduler_buffer[2] = Common::swap32(new_code);
 }
 
+NWC24::ErrorCode NetKDRequestDevice::KDCheckMail(u32* _mail_flag, u32* _interval)
```

Comment:
Why is that `_` prefix here? It looks really odd.

Reply:
I did that because I made a `mail_flag` var inside the function. You are right though, I should change it to not have the `_`